RepoDb.Extensions/RepoDb.SqlServer.BulkOperations/RepoDb.SqlServer.BulkOperations/Compiler.cs

```diff
@@ -283,19 +283,15 @@ private static class PropertyGetterFuncCache<TEntity, TResult>
             /// <returns></returns>
             public static Func<TEntity, TResult> GetFunc(ClassProperty classProperty)
             {
-                var func = (Func<TEntity, TResult>)null;
-                if (cache.TryGetValue(classProperty.GetHashCode(), out func) == false)
+                if (cache.TryGetValue(classProperty.GetHashCode(), out var func) == false)
```

Comment:
We have too many like this within the library. I think, it is time to refactor the code written in those as well. Thanks for this